chromium_src/chrome/browser/sync/profile_sync_service_factory.cc

```diff
@@ -3,16 +3,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "brave/browser/signin/brave_identity_manager_factory.h"
 #include "brave/browser/sync/brave_profile_sync_service_delegate.h"
 #include "brave/components/sync/driver/brave_sync_profile_sync_service.h"
 #include "chrome/browser/sync/device_info_sync_service_factory.h"
 
-#define BRAVE_BUILD_SERVICE_INSTANCE_FOR             \
-  std::make_unique<syncer::BraveProfileSyncService>( \
-      std::move(init_params),                        \
+#define IdentityManagerFactory BraveIdentityManagerFactory
+
+#define BRAVE_BUILD_SERVICE_INSTANCE_FOR                         \
+  std::make_unique<syncer::BraveProfileSyncService>(             \
+      std::move(init_params),                                    \
       std::make_unique<syncer::BraveProfileSyncServiceDelegate>( \
           DeviceInfoSyncServiceFactory::GetForProfile(profile)));
 
+// Excluding src/chrome/browser/signin/identity_manager_factory.h, but this is
+// OK, because we are including brave_identity_manager_factory.h
+#define CHROME_BROWSER_SIGNIN_IDENTITY_MANAGER_FACTORY_H_
```

Comment:
without this define I had a lots of errors because `#define IdentityManagerFactory BraveIdentityManagerFactory` caused substitution in `#include "components/signin/......../identity_manager_factory.h"` , or for some other includes which are using `IdentityManagerFactory`, but now this doesn't happen. Will remove.

Reply:
removed in 765f842236ec7c12618c4aa6844a3b954ca88ffe

Reply:
next time you can include `chrome/browser/signin/identity_manager_factory.h` directly before `#define` so it won't substitute something that we don't intend to in the header

Reply:
thanks :+1: 

at that time it was something more complex, I recall it was a nested include